htdocs/index.php

```diff
@@ -26,6 +26,7 @@
 
 define('NOCSRFCHECK', 1); // This is main home and login page. We must be able to go on it from another web site.
 
+require_once dirname(__DIR__).'/vendor/autoload.php';
```

Comment:
This dir and file does not exists.
All hardcoded link in code must refer to directories that exists in the git repository so any installation using git clone, or packager based on rpm or dpkg or on .exe for windows will work.